static/css/style.css

```diff
@@ -299,6 +336,58 @@ h1.table_titles{
     z-index: 1021;
 }
 
+.team-card {
+    min-height: 380px;
+}
+
+.card-title .team-name:after {
```

Comment:
CSS3 standard is `::after` instead of `:after`.

Reply:
Changed as suggested.